Merge Conflict/Assets/Scripts/Order/Calculate.cs

```diff
@@ -0,0 +1,85 @@
+/**********************************************************************************************************************
+Name:           Calculate
+Description:    Contains all sub-calculations to calculate the probabilitie for the stages from an given component. 
+                It is needed to generate an new order.
+Author(s):      Markus Haubold
+Date:           2024-03-20
+Version:        V1.0
+TODO:           - 
+**********************************************************************************************************************/
+
+using System;
+using System.Linq;
+
+public static class Calculate
```

Comment:
I used Calculate to get good combinations together with the methods.
Eg.: Calculate.ScaleDistance()

I thought it would be better to make an extra class to separate the calculation from the order generation.